Resources/Private/Templates/Module/Index.html

```diff
@@ -16,8 +16,8 @@
 
             data-login-endpoint="{f:uri.action(action: 'authenticate', controller: 'Login', package: 'Neos.Neos', format: 'json', absolute: true)}"
 
-            data-module-uri="{f:uri.action(action:'index', controller:'module', package:'Sitegeist.Monocle', absolute: true)}"
-            data-fullscreen-uri="{f:uri.action(action:'index', controller:'module', package:'Sitegeist.Monocle', absolute: 1)}"
+            data-module-uri="{f:uri.action(action:'index', controller:'module', package:'Sitegeist.Monocle')}"
```

Comment:
I am not entirely sure wether there was a deeper reason why this url was absolute. The change needs this to be relative to the root.